plugins/inputs/redis/redis.go

```diff
@@ -248,6 +248,12 @@ func gatherInfoOutput(
 				gatherKeyspaceLine(name, kline, acc, tags)
 				continue
 			}
+			if section == "Replication" && strings.HasPrefix(name, "slave") {
```

Comment:
This pulls in some other lines that begin with `slave`, and so they no longer get processed.  We should add a unit test for slave redis.